test/Conversion/TorchToLinalg/basic.mlir

```diff
@@ -67,3 +67,17 @@ func @integer_extract(%arg0: !torch.vtensor<[],si64>) -> !torch.int {
   %0 = torch.aten.Int.Tensor %arg0 : !torch.vtensor<[],si64> -> !torch.int                   
   return %0 : !torch.int                 
 }                       
+
+// -----
+
+// CHECK:    func @ntt(%[[IN:.*]]: !torch.int) -> !torch.vtensor<[],si64> {
+// CHECK:      %[[A:.*]] = torch_c.to_i64 %[[IN]]
```

Comment:
It's better to replace the `A`, `B`, `C` to more meaningful names like CST, INIT, FILL